src/website/src/app/documentation/demos/alert/alerts.demo.html

```diff
@@ -421,28 +421,25 @@ <h2>Accessibility</h2>
                 </clr-alert-item>
             </clr-alert>
             <p>Accessibility problems related to using actionable controls inside dynamically generated alerts:</p>
-            <ul class="list">
-                <li>
-                    They are announced as part of the alert message, which is out of context and may be confusing. For example, the following alert will be announced as "success acknowledge":
-                    <div class="clr-row">
-                        <div class="clr-col-12 clr-col-md-6">
-                            <clr-alert clrAlertType="success" [clrAlertClosable]="false">
-                                <clr-alert-item>
-                                    <span class="alert-text">
-                                        Success
-                                    </span>
-                                    <div class="alert-actions">
-                                        <a class="alert-action" href="javascript://">Acknowledge</a>
-                                    </div>
-                                </clr-alert-item>
-                            </clr-alert>
+
+            <div class="alert alert-warning">
+                <div class="alert-items">
+                    <div class="alert-item static">
+                        <div class="alert-icon-wrapper">
+                            <clr-icon class="alert-icon" shape="exclamation-triangle"></clr-icon>
                         </div>
+                        <span class="alert-text">
+                            As from V3 we no longer provide <code class="clr-code">aria-live</code> support as part of the
```

Comment:
```Since v3, we no longer handle setting `aria-live` and announcing the message for you by default. Based on the application use case, you can use the new `ClrAriaLiveService` to make announcements when they make sense for a user to hear about updates or loading status changes.```